### ISSUE TITLE
package ‘htmltab’ is not available (for R version 3.6.3)

### ISSUE THREAD

**Issue by herssede** [crubba/htmltab#23]
Getting error message trying to install. (Using RStudio)

**Comment by burgerga** [comment]
Yes, it was removed from CRAN because issues were not fixed in time (see https://cran.r-project.org/web/packages/htmltab/index.html)
The last commit on this repository is from 5 years ago, and although @crubba pushed the v0.7.1.1 release to CRAN, the commits were not pushed to Github.

You can install the package from the CRAN archive (https://cran.r-project.org/src/contrib/Archive/htmltab/) or you can install from this github or one of the forks (https://github.com/crubba/htmltab/network). My fork (https://github.com/burgerga/htmltab) includes the v0.7.1.1 code (which I extracted from the version on CRAN), if you want to install it use 

```r
# install.packages("remotes")
remotes::install_github("burgerga/htmltab")
```

@barclay-osborne, @feuerbach Do you want to join forces working on a fork?

**Comment by UnkindPartition** [comment]
Yeah, definitely. I don't have the bandwidth to be a day-to-day maintainer, but I'm happy to help get it back on CRAN and then be a backup maintainer.

**Comment by burgerga** [comment]
I don't want to push a fork just yet
@crubba can you give us collaborator status on this repo? That way we can keep the issues, and the pull requests

**Comment by NathanLazarus** [comment]
Thanks all, I have some old code that depends on htmltab and I'd love to see it back on CRAN and compatible with R 4.0.x.

**Comment by burgerga** [comment]
@UnkindPartition I just published some additional fixes to https://github.com/htmltab/htmltab, and invited you to join. Maybe you can reapply your pull request there?
Then we can push it to CRAN again.

**Comment by burgerga** [comment]
It's on CRAN again :)

**Comment by UnkindPartition** [comment]
Great job @burgerga, thanks!
